docs/path.md

```diff
@@ -0,0 +1,45 @@
+# Allow Path and Ignore Path
+
+## Description
+
+Paths that are not provided by `--allow-path` will be rejected.
+Paths that are provided by `--ignore-path` won't be rejected.
+It is only possible to define one of both.
```

Comment:
I think this is a bit confusing. How about:

```suggestion
If `--allow-paths` is set, only the requests to these paths will be considered for proxying, the rest will be rejected.
If `--ignore-paths` is set, requests to these paths will be passed upstream without any authorization.

Only one of `--allow-paths`/`--ignore-paths` is allowed to be set at a time.
```

Note that the README says `allow/ignore-paths` whereas here you're describing it as `allow/ignore-path`